test/ex_webrtc/rtp/jitter_buffer/realtime_test.exs

```diff
@@ -0,0 +1,117 @@
+defmodule ExWebRTC.RTP.JitterBuffer.RealtimeTest do
+  use ExUnit.Case
+
+  alias ExWebRTC.RTP.{JitterBuffer, PacketFactory}
+  alias ExRTP.Packet
+
+  @seq_number_limit 65_536
+
+  defmodule PacketSource do
+    @moduledoc false
+    use GenServer
+
+    @seq_number_limit 65_536
+
+    @impl true
+    def init(state) do
+      {:ok, state, {:continue, :after_init}}
+    end
+
+    @impl true
+    def handle_continue(
+          :after_init,
+          %{
+            packet_delay_ms: delay_ms,
+            packet_num: packet_num,
+            max_latency: max_latency
+          } = state
+        ) do
+      now = System.monotonic_time(:millisecond)
+
+      1..packet_num
+      |> Enum.each(fn n ->
+        time =
+          cond do
+            # Delay less than max latency
+            rem(n, 15) == 0 -> n * delay_ms + div(max_latency, 2)
```

Comment:
This block of code schedules sending *all* of the packets.

`n * delay_ms` is the base time when the packet is expected to arrive (when it would be sent, were there no delay), while `+ div(max_latency, 2)` is the delay